cloud/ec2.go

```diff
@@ -62,6 +62,9 @@ type EC2ProviderSettings struct {
 	// SecurityGroupIDs is a list of security group IDs.
 	SecurityGroupIDs []string `mapstructure:"security_group_ids" json:"security_group_ids,omitempty" bson:"security_group_ids,omitempty"`
 
+	// IAMInstanceProfileArn is the Amazon Resource Name (ARN) of the instance profile.
+	IAMInstanceProfileArn string `mapstructure:"iam_instance_profile_arn,omitempty" json:"iam_instance_profile_arn,omitempty"  bson:"iam_instance_profile_arn,omitempty"`
```

Comment:
ARN is an acronym for Amazon Resource Name so this should be `IAMInstanceProfileARN` for casing consistency.